.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
       go: 1.12.x
       script:
         - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
-        - golangci-lint run --disable errcheck --enable golint; 
+        - golangci-lint run --disable errcheck --enable golint --exclude 'SA4008'; 
```

Comment:
Can you explain what SA4008 is? Why is it excluded?

Reply:
In the file, there is this section of the code of listening where you return and statictest is not happy because usually in ur for loop, u will increment but you did not just because you will forever stuck in there. This is to skip that special case because without return or break in that for loop. It will never exit.

Reply:
I have put comments on that line of code in this pr too.

Reply:
I see. After merging, I will try to update the example to see if we can avoid the error. If we can, I will remove this flag.